testlist/sim.dat

```diff
@@ -1,5 +1,6 @@
 /sim
 -sim:cxxtest
+-sim:nshcromfs
```

Comment:
this will disable nshcromfs build, does it your intention?

Reply:
No